tpdatasrc/tpgamefiles/scr/tpModifiers/stormlord.py

```diff
@@ -203,6 +204,38 @@ def pythonActionPerformStorm(attachee, args, evt_obj):
 #stormlordSLA.AddHook(ET_OnD20PythonActionPerform, pythonActionStormId, pythonActionPerformStorm, ())
 
 
+#### Stormlord Particles ####
+
+def particlesRadial(attachee, args, evt_obj):
+    print "particlesRadial Hook"
+    if args.get_arg(0):
+        toggleEffectId = tpdp.RadialMenuEntryPythonAction("Deactivate Crackling Aura", D20A_PYTHON_ACTION, pythonActionCracklingAuraID, 0, "TAG_STORMLORDS")
+    else:
+        toggleEffectId = tpdp.RadialMenuEntryPythonAction("Activate Crackling Aura", D20A_PYTHON_ACTION, pythonActionCracklingAuraID, 1, "TAG_STORMLORDS")
+    toggleEffectId.add_child_to_standard(attachee, tpdp.RadialMenuStandardNode.Class)
+    return 0
+
+def toggleCracklingAura(attachee, args, evt_obj):
```

Comment:
Can't wait to check it out :D

Question: does it rekindle after loading save/switching maps?
Also perhaps instead of having it an explicit action make it start up when entering combat, and the radial option would be a simple checkbox then.

Reply:
Hmm forgot about the teleport. Fixed and pushed :)

I don't know which version is better, combat only or a plain toggle. This up to you to decide :)

Reply:
I think having it on just for combat would make it more special. Having it on constantly would probably be annoying, as would having to constantly toggle it manually.